src/main/java/use_cases/display_availability_timeblocks/DisplayAvailabilityTimeBlockInputBoundary.java

```diff
@@ -0,0 +1,17 @@
+package use_cases.display_availability_timeblocks;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+/**
+ * An input boundary interface that is implemented by the DisplayTimeBlockInteractor class.
+ *
+ * Created: 11/22/2022
+ * Last updated: 11/22/2022
+ *
+ * @author Amir Bare
+ */
+
+public interface DisplayAvailabilityTimeBlockInputBoundary {
+    DisplayAvailabilityTimeBlockModel displayAvailabilityTimeBlock();
```

Comment:
We want to make sure we're documenting _everything_! This includes both classes, **and** methods, especially on the interface layers. Imagine it like this: we want to make it so that, if someone else has to use these interfaces' code, they can hover over the method in IntelliJ and be told _exactly_ what the method is doing, without needing to actually look at the source code. David's use case and my entities layer should be of help in finding direction with that. It's not enough, for me, to want to delay this PR any longer, especially with the knowledge that this UI portion of the project is still a work-in-progress, so I know it will be covered eventually. Just something for the future!